src/main.cpp

```diff
@@ -11,13 +10,13 @@ namespace qn = qindesign::network; //setup of qn namespace
 qn::EthernetUDP udp; //setup of qn namespace
 
 /* Functions */
-void configure_charging();
+
 
 void setup() {
   //Serial.begin(115200); //I dont know if this is the baudrate I also don't know if this is even needed
   qn::Ethernet.begin(); //begins QNEthernet
+  
 }
 
 void loop() {
-  charge_cycle();
 }
```

Comment:
Your main should be:
1) Initializing a charging system
2) Initializing Tasks (see HT_SCHED implementation in VCF)
3) Beginning serial if necessary
4) Initializing an instance of Scheduler
5) Calling Scheduler.run() in loop